### ISSUE TITLE
The Fix button on the Alerts with pager control lacks valid visual indication of focus due to the color contrast with the background color

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4651]
*chlane - @gkhadK found that when the user agent's default focus indicator is obscured by the styling of another element, the author needs to supply a custom visual focus indication or remove the styling that is obscuring the visual focus indication. See https://www.w3.org/WAI/WCAG21/Techniques/failures/F78

[Clarity Version]
3.1.1

[Test URL]
http://chrislane.info/clarity3demo1/

[WCAG 2.1 Guideline]
2.4.7
(Level A)

[OS]
Windows 10

[User Agents/Assistive Technologies]
Firefox 75.0, Switch

[Component]
Alerts

[Description]
The Fix button on the Alerts with pager control lacks valid visual indication of focus due to the color contrast with the background color

[User Impact]
Switch users have difficulty tracking their keyboard focus because it is hard to visually see that their focus is on the button due to the color contrast. This also makes it difficult to activate the button because it is hard to see that their focus is on the button. 

[Steps to Reproduce]
See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

1. Using a switch, try navigating to and activating the Fix button under the Alerts with pager control heading. 
2. Notice that there is no visual indication of focus in order to know which button you are on. 

[Recommendation]
While an outline rule is a part of the default Firefox focus indicator, the existing button border may prevent the outline from being easily seen. Applying a positive outline-offset will ensure that the focus indicator will be more readily seen.	&quot;.alert-app-level.alert-info .alert-action.btn, .alert-app-level.alert-info .dropdown-toggle.btn {
    ...
    outline-offset: 3px
}&quot;

[Reporter Email] 
daman@damanw.com
